drafts/from-borderlands-to-cornerlands.md

```diff
@@ -0,0 +1,185 @@
+---
+title: From Borderlands to Cornerlands - Why border-radius is wrong?
+domain: software-engineering-corner.hashnode.dev
+tags: web-development, css, css3
+cover: https://cdn.hashnode.com/res/hashnode/image/upload/v1701859904085/Txlm1bi%5FG.jpg?w=1600&h=840&fit=crop&crop=entropy&auto=compress,format&format=webp
+publishAs: juliocastrodev
+hideFromHashnodeCommunity: false
+ignorePost: true
+---
+
+Why isn't my div centered? Why does it overflow? Why do I always have to override a lot of rules to make CSS less unpredictable?
```

Comment:
I would put 'div' in code styling too, similar to box-sizing, border-box etc. later on.